src/test/java/seedu/address/model/player/EnemyTest.java

```diff
@@ -1,12 +1,28 @@
 package seedu.address.model.player;
 
+import static org.assertj.core.api.Assertions.not;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertTrue;
```

Comment:
`assertTrue` is no longer used. This import is not required here, according to CheckStyle.